tests/tests/snapshot.rs

```diff
@@ -28,10 +28,10 @@ async fn loads_snapshot() {
     // setup config
     let starting_state = StateConfig {
         last_block: Some(LastBlockConfig {
-            block_height: (u32::MAX - 1).into(),
+            block_height: (u32::MAX - 2).into(),
```

Comment:
Why do we need to change this file?

Reply:
Because the code internally calls `next_block` now. If it's `u32::Max` (as it becomes) then `next_block` will panic.

Reply:
Then test shouldn't produce any blocks, maybe we need to disable block producer

Reply:
Adding `Trigger::Never` doesn't seem to fix the issue; the producer is still being run.